From 4e97130b71a6c4318b7aae8d79cce3f53b4bd17d Mon Sep 17 00:00:00 2001
Message-Id: <4e97130b71a6c4318b7aae8d79cce3f53b4bd17d.1490187394.git.asathoor@gmail.com>
From: asathoor <per@multimusen.dk>
Date: Mon, 26 Oct 2015 21:59:03 +0000
Subject: [PATCH] simple gallery

---
 scandir.php | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/scandir.php b/scandir.php
new file mode 100644
index 0000000..7d87a42
--- /dev/null
+++ b/scandir.php
@@ -0,0 +1,19 @@
+<h1>The most simple gallery</h1>
+
+<?php
+$dir    = 'images';
+$files1 = scandir($dir); // . .. foerst
+$files2 = scandir($dir, 1); // . .. sidst
+
+/*remove (=unset)  . .. from files1*/
+unset($files1[0],$files1[1]);
+
+echo "<div id='gallery'>";
+
+/*loop out object content*/
+foreach ($files1 as $img) {
+	echo "<img src='images/$img' width='150' style='float:left' />";
+}
+
+echo "</div>";
+?>
-- 
2.9.3

